x/name/types/msgs_test.go

```diff
@@ -0,0 +1,59 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func TestMsgCreateRootNameRequestGetSignBytes(t *testing.T) {
```

Comment:
I'm not saying these need to be refactored, but for future new tests, consider using [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests). Here's a `ValidateBasic` example from our metadata module: https://github.com/provenance-io/provenance/blob/bcc58b01f5ca00dacb2fb908fee9f374bef5eaf0/x/metadata/types/msg_test.go#L99-L148